addons/l10n_anz_ubl_pint/models/account_edi_xml_pint_anz.py

```diff
@@ -0,0 +1,118 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+
+
+class AccountEdiXmlUBLPINTANZ(models.AbstractModel):
+    _inherit = "account.edi.xml.ubl_bis3"
+    _name = 'account.edi.xml.pint_anz'
+    _description = "Australia & New Zealand implementation of Peppol International (PINT) model for Billing"
+    """
+    Pint is a standard for International Billing from Peppol. It is based on Peppol BIS Billing 3.
+    It serves as a base for per-country specialization, while keeping a standard core for data being used
+    across countries. This is not meant to be used directly, but rather to be extended by country-specific modules.
+
+    The ANZ PINT format is the Australia & New Zealand implementation of PINT.
+
+    * PINT Official documentation: https://docs.peppol.eu/poac/pint/pint/
+    * PINT ANZ Official documentation: https://docs.peppol.eu/poac/aunz/pint-aunz/
+    """
+
+    def _export_invoice_filename(self, invoice):
+        # EXTENDS account_edi_ubl_cii
+        return f"{invoice.name.replace('/', '_')}_pint_anz.xml"
+
+    def _get_partner_party_vals(self, partner, role):
+        # EXTENDS account.edi.xml.ubl_bis3
+        vals = super()._get_partner_party_vals(partner, role)
+
+        if partner.country_code == 'AU' and partner.vat:
+            vals['endpoint_id'] = partner.vat.replace(" ", "")
+        if partner.country_code == 'NZ':
+            vals['endpoint_id'] = partner.company_registry
```

Comment:
It's been a while, I don't remember :sweat:
I think i took this from the ubl module. I think it may be wrong, as the eas for au is the brn :thinking: 
For nz there are no options in the eas from what i found.